packages/providers/codegen.json

```diff
@@ -28,6 +28,37 @@
         "nonOptionalTypename": true,
         "typesPrefix": "Gql"
       }
+    },
+    "./src/__generated__/subscriptions.ts": {
+      "documents": "./src/subscriptions.graphql",
+      "plugins": [
+        { "typescript": {} },
+        { "typescript-operations": {} },
+        {
+          "typescript-generic-sdk": {
+            "documentMode": "string"
+          }
+        }
+      ],
+      "config": {
+        "scalars": {
+          "DateTime": "string",
+          "HexString": "string",
+          "Bytes32": "string",
+          "UtxoId": "string",
+          "U64": "string",
+          "Address": "string",
+          "BlockId": "string",
+          "TransactionId": "string",
+          "AssetId": "string",
+          "ContractId": "string",
+          "Salt": "string",
+          "Nonce": "string"
+        },
+        "useTypeImports": true,
+        "nonOptionalTypename": true,
+        "typesPrefix": "Gql"
+      }
```

Comment:
Curious why you created separate files just for the subscriptions?

Reply:
I separated them because they're constructed differently, use different libraries, etc. However, I've got an idea on how to put them into one file and I'll investigate it.

Reply:
New implementation has no need for the `subscriptions.graphql` file.